data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -38,6 +38,20 @@ OPENSHIFT_HYPERKUBE_IMAGE=$(podman run --quiet --rm ${release} image hyperkube)
 
 CLUSTER_BOOTSTRAP_IMAGE=$(podman run --quiet --rm ${release} image cluster-bootstrap)
 
+# Verify at this point we have no images from docker.io or gcr.io
```

Comment:
I would not include a `test` in the code shipped to users. It should be e2e test kind of thing.

Reply:
Hmm.  Today the e2e tests start after the bootstrap node has been destroyed...

But I could imagine customer use cases where the bootstrap machine happens to have some of their custom images on it.  So we probably can't ship this test as is for that reason too.

Reply:
So I guess if we really want to test this what we need to do is change CI to disallow the bootstrap node from pulling from `gcr.io` or any other registries by...hum, injecting some Ignition?